resources/views/graph.blade.php

```diff
@@ -1,3 +1,10 @@
+<!--This is the result page for the website of the filled in workflows. The result page shows the result of the Evidencio API call.
+This is represented in a chart, the user is able to choose the result to be diaplayed in different graphical chats by clicking on different buttons.
+The will be also a text result if any was included in the Evidencio API call.
+Users are also able to download a pdf of the results-->
+
+
+<!--post request to Evidencio API with the answers of the workflow page. This will return an array with the result and their parameters -->
```

Comment:
The same as above, in the rendered HTML you would see the comment but no PHP code.